content/docs/reference-react-dom.md

```diff
@@ -62,15 +62,15 @@ If the optional callback is provided, it will be executed after the component is
+Jika atribut dari sebuah elemen atau isi text berbeda antara *server* dan klien (contohnya, sebuah *timestamp*), Anda dapat membungkam *warning* dengan menambahkan `suppressHydrationWarning={true}` di elemen tersebut. Ini hanya dapat dilakukan pada kedalaman satu tingkat, dan dimaksudkan sebagai jalan keluar. Jangan disalahgunakan. Kecuali isi teksnya, React tidak akan mencoba untuk memperbaiki hal ini, jadi ini mungkin akan tetap tidak konsekuen sampai diperbarui di masa depan.
 
-If you intentionally need to render something different on the server and the client, you can do a two-pass rendering. Components that render something different on the client can read a state variable like `this.state.isClient`, which you can set to `true` in `componentDidMount()`. This way the initial render pass will render the same content as the server, avoiding mismatches, but an additional pass will happen synchronously right after hydration. Note that this approach will make your components slower because they have to render twice, so use it with caution.
+Jika Anda sengaja perlu untuk me-*render* sesuatu yang berbeda antara *server* dan klien, Anda dapat melakukan *two-pass rendering*. Komponen-komponen yang me-*render* sesuatu yang berbeda di klien dapat membaca sebuah *state variable* seperti `this.state.isClient`, dimana Anda dapat menetapkannya menjadi `true` di `componentDidMount()`. Dengan cara ini *render pass* pertama akan me-*render* isi yang sama dengan *server*, menghindari ketidakcocokan, tetapi sebuah *pass* tambahan akan terjadi secara sinkron langsung setelah *hidrasi*. Perhatikan bahwa cara ini akan mengakibatkan komponen-komponen Anda lebih lambat karena mereka harus me-*render* dua kali, jadi gunakan dengan hati-hati.
 
-Remember to be mindful of user experience on slow connections. The JavaScript code may load significantly later than the initial HTML render, so if you render something different in the client-only pass, the transition can be jarring. However, if executed well, it may be beneficial to render a "shell" of the application on the server, and only show some of the extra widgets on the client. To learn how to do this without getting the markup mismatch issues, refer to the explanation in the previous paragraph.
+Ingat untuk berhati-hati dalam hal *user experience* pada saat koneksi lambat. Kode JavaScript mungkin akan dimuat lebih lama dari *render* HTML pertama, jadi jika Anda me-*render* sesuatu yang beda di *client-only pass*, transisinya bisa kasar. Meskipun begitu, jika dijalankan dengan baik, mungkin menguntungkan jika me-*render* cangkang dari aplikasi di *server*, dan hanya memperlihatkan beberapa *widget* ekstra di klien. Untuk belajar bagaimana melakikan ini tanpa mengalami masalah ketidakcocokan *markup*, lihat penjelasan di paragraf sebelumnya.
```

Comment:
Initial HTML render -> *render* HTML pada saat pertama kali (?)
*client-only pass* -> memang gak di translate juga ya?
"shell" -> kayanya lebih baik dipertahankan, daripada kata cangkang.
melakikan -> melakukan (typo)

Reply:
Ingatlah untuk berhati-hati dalam hal  pengalaman pengguna (*user experience*) pada saat koneksi lambat. Kode JavaScript mungkin akan dimuat lebih lambat dari saat *render* HTML pertama kali, jadi jika Anda me-*render* sesuatu yang hanya ada di klien saja, transisinya bisa jadi kasar. Meskipun begitu, jika dieksekusi dengan baik, mungkin akan bermanfaat untuk me-*render* "*shell*" dari aplikasi di server, dan hanya menunjukkan beberapa  *widget* ekstra di klien. Untuk belajar mengenai bagaimana melakukan ini tanpa mengalami masalah ketidakcocokan *markup*, silahkan lihat penjelasan di paragraf sebelumnya.

What do you think @mazipan @anggras ?

Reply:
Cocok, lebih mudah dipahami konteksnya